src/database/client.rs

```diff
@@ -68,6 +77,14 @@ impl PgDbClient {
         update_mergeable_states_by_base_branch(&self.pool, repo, base_branch, mergeable_state).await
     }
 
+    pub async fn get_pull_requests_by_base_branch(
```

Comment:
```suggestion
    pub async fn get_nonclosed_pull_requests_by_base_branch(
```
Otherwise the name of the function kind of lies (it doesn't return all PRs by base branch).